azuread/README.md

```diff
@@ -0,0 +1,41 @@
+# Azure AD / Entra ID App Function HTTP Trigger Example
+This C# script example demonstrates how to call the SGNL Access Service from an Azure App Function HTTP Trigger. This app function and trigger is then configured as part of an custom authentication extension.
+
+
+# Prerequisites
+
+1. You have a SGNL environment up and running.
+ 
+2. You have configured connectors to ingest data and have defined the appropriate relationship mappings.
+ 
+3. You have created policy snippets and policy for evaluation.
+ 
+4. You have created an protected application to test. You can test with this [example](https://jwt.ms/) provided by Microsoft.
+ 
+5. You will need Git to clone the example repository. You can follow the steps to install Git [here](https://github.com/git-guides/install-git).
+
+
+See our [Help Guides](https://support.sgnl.ai) for steps on configuring data sources and policies.
+
+
+## Steps For Configuring the custom authentication extension.
+
+
+1. Clone the SGNL examples repo using Git from https://github.com/SGNL-ai/examples.git.
+2. Login to your Azure portal.
+3. Follow the steps found [here](https://learn.microsoft.com/en-us/azure/active-directory/develop/custom-extension-get-started?tabs=azure-portal%2Chttp).
+4. Replace the default C# script in the example in step 3, with the run.csx C# code from the SGNL examples repository.
+
+
+## Steps For Testing The Example
+
+1. Once your configuration is complete, you can build your login URL for initiating the Azure AD IDP authentication flow. See the example URL below:
```

Comment:
I'd suggest just using the myapps portal - this seems more complex than necessary.

Reply:
It was actually pretty easy to implement.